cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -174,42 +177,71 @@ define(
                             // 2.0, the latest version of jasmine-ajax (mock-ajax.js) does have the
                             // necessary support.
 
-                            _.each(
-                                [
-                                    {
-                                        desc: 'completion',
-                                        responseStatus: 204,
-                                        statusText: ActiveVideoUpload.STATUS_COMPLETED,
-                                        progressValue: 1,
-                                        presentClass: 'success',
-                                        absentClass: 'error'
-                                    },
-                                    {
-                                        desc: 'failure',
-                                        responseStatus: 500,
-                                        statusText: ActiveVideoUpload.STATUS_FAILED,
-                                        progressValue: 0,
-                                        presentClass: 'error',
-                                        absentClass: 'success'
-                                    }
-                                ],
+                            _.each([true, false],
+                                function(viewRefresh) {
+                                    var refreshDescription = viewRefresh ? ' (refreshed)' : ' (not refreshed)';
+                                    _.each(
```

Comment:
I don't think we would need `_.each` to set `subCases` here now.  You can directly set `subCases` like
```
subCases = [
              {
                 desc: 'completion' + refreshDescription,
                 responseStatus: 204,
...
```